apps/prairielearn/src/lib/authn.js

```diff
@@ -81,23 +81,26 @@ export async function loadUser(req, res, authnParams, optionsParams = {}) {
       user_id: user_id,
       authn_provider_name: authnParams.provider || null,
     };
-    var pl_authn = generateSignedToken(tokenData, config.secretKey);
-    res.cookie('pl_authn', pl_authn, {
+    var pl2_authn = generateSignedToken(tokenData, config.secretKey);
+    res.cookie('pl2_authn', pl2_authn, {
```

Comment:
I very much don't like `pl2_*` as a permanent prefix for our cookies, though it's not the worst thing in the world. Two thoughts:

- We could instead use `pl.` as the prefix (e.g. `pl.authn`).
- We could instead use `prairielearn_` as the prefix (e.g. `prairielearn_authn`).
- We could do a reverse migration back to `pl_*` after fully deploying `pl2_*`. My concern here is that we could end up with a client that still has an old `pl_` cookie that's scoped to the wrong domain by the time we start the reverse migration. We could probably avoid that by being careful with timing and cookie expiration dates?

Reply:
We decided we'll do a reverse migration at some point in the future.